packages/editor/src/editor.spec.ts

```diff
@@ -4,14 +4,32 @@ import { Duplex, PassThrough } from 'stream';
 import Nanobus from 'nanobus';
 import path from 'path';
 import { promises as fs } from 'fs';
+import { promisify } from 'util';
+import rimraf from 'rimraf';
 import sinon from 'ts-sinon';
 import sinonChai from 'sinon-chai';
 
 import { Editor } from './editor';
-import { eventually } from '../../../testing/eventually';
 
 chai.use(sinonChai);
 
+const fakeExternalEditor = async(
+  { base, name, flags = '', output }: { base: string, name: string, flags?: string, output: string }
+): Promise<string> => {
+  const tmpDoc = path.join(base, name);
+  const script = `(async () => {
+    const tmpDoc = process.argv[process.argv.length - 1];
+    const { promises: { writeFile } } = require('fs');
+
+    await writeFile(tmpDoc, \`${output}\`, { mode: 0o600 });
```

Comment:
```suggestion
    await writeFile(tmpDoc, ${JSON.stringify(output)}, { mode: 0o600 });
```

So that this doesn’t break if `output` contains a backtick or escape sequences?

Reply:
Well, I don't have these cases in my tests, so it doesn't :) If we want to cover such cases in tests I can replace it with JSON.stringify.